Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apk add --no-cache tzdata
 
 WORKDIR /app
 
-COPY --from=builder /app/entrypoint.sh /app/package.json /app/package-lock.json /app/tsconfig.json ./
+COPY --from=builder /app/ormconfig.js /app/scripts/entrypoint.sh /app/package.json /app/package-lock.json /app/tsconfig.json ./
```

Comment:
ormconfig.js should not be copied, if we are still using environment variables for TypeORM.

Reply:
Wont it just end up resolving to the ENV variables if it gets both a configuration file and ENV exports? Atleast thats what I am used to from Entity Framework Core in .NET Core so I figured it would be OK, but I will omit the file as I trust your input on this matter. :)

Reply:
It is not okay because ormconfig.js is non existent file and there is nothing to do with .NET